ui/src/gen/schema/projects.kargo.akuity.io_v1alpha1.json

```diff
@@ -31,9 +31,6 @@
                 "type": "string"
               }
             },
-            "required": [
-              "stage"
-            ],
```

Comment:
Similar comment made about CRD definitions. This looks like it shouldn't be removed.

I am wondering if `//+kubebuilder:validation:Required` is being ignored?

Reply:
> I didn't noticed that the omitempty has presedence over kubebuilder's
Required marker

That has tripped me up before as well, as recently as yesterday.